process/smartContract/builtInFunctions/esdtTransfer.go

```diff
@@ -96,24 +102,24 @@ func (e *esdtTransfer) addToESDTBalance(userAcnt state.UserAccountHandler, key [
 	return nil
 }
 
-func (e *esdtTransfer) getESDTDataFromKey(userAcnt state.UserAccountHandler, key []byte) *ESDigitalToken {
+func (e *esdtTransfer) getESDTDataFromKey(userAcnt state.UserAccountHandler, key []byte) (*ESDigitalToken, error) {
 	esdtData := &ESDigitalToken{Value: big.NewInt(0)}
 	marshalledData, err := userAcnt.DataTrieTracker().RetrieveValue(key)
 	if err != nil {
 		log.Debug("getESDTDataFromKey retrieve value error", "error", err)
-		return esdtData
+		return esdtData, nil
 	}
 
 	err = e.marshalizer.Unmarshal(esdtData, marshalledData)
 	if err != nil {
 		log.Debug("getESDTDataFromKey unmarshal error", "error", err)
```

Comment:
This log could be removed now, as would be printed in the caller side

Reply:
done